apps/notifier/src/pull-listener.ts

```diff
@@ -1,7 +1,13 @@
 import { logger, PubSubClient } from '@bike4life/commons'
 import { Message } from '@google-cloud/pubsub'
+import  sgMail = require('@sendgrid/mail')
```

Comment:
We shouldn't use the `require()` expression, as it's not the proper way to import dependencies with typescript 🤔